tests/test_pipeline_azure_streaming.py

```
@@ -0,0 +1,148 @@
+        self.subscription = subscription
+        self.region = region
+        self.properties = {}
+        self.speech_recognition_language = None
+        self.output_format = None
+
+    def set_property(self, key, value):
+        self.properties[key] = value
+
+
+class _AudioStreamFormat:
+    created = []
```

Ruff (0.15.18)

[warning] Mutable default value for class attribute

(RUF012)